features/kernel/src/main/feature/feature.xml

```diff
@@ -64,14 +64,15 @@
 
     <!-- Override the pax-web pax-http-whiteboard feature and remove the vulnerable pax-web-jsp bundle-->
     <feature name="pax-http-whiteboard" description="Provide HTTP Whiteboard pattern support" version="${pax.web.version}_1">
-        <feature version="[6.0,6.1)">pax-http</feature>
+        <feature version="[7.2,7.3)">pax-http</feature>
```

Comment:
This feature override was just so we could remove the vulnerable version of pax-web-jsp. Version 7.2.3 is fine according to sonatypes so we should be able to remove this feature.